src/System.Windows.Forms/src/System/Windows/Forms/ToolStripDropDownButton.ToolStripDropDownButtonInternalLayout.cs

```diff
@@ -12,44 +10,45 @@ public partial class ToolStripDropDownButton
     {
         private protected class ToolStripDropDownButtonInternalLayout : ToolStripItemInternalLayout
         {
-            private ToolStripDropDownButton ownerItem;
-            private static readonly Size dropDownArrowSizeUnscaled = new Size(5, 3);
-            private static Size dropDownArrowSize = dropDownArrowSizeUnscaled;
+            private ToolStripDropDownButton _ownerItem;
+            private static readonly Size s_dropDownArrowSizeUnscaled = new Size(5, 3);
```

Comment:
```suggestion
            private static readonly Size s_dropDownArrowSizeUnscaled = new (5, 3);
```

Reply:
In that case, it'd be this :)
```suggestion
            private static readonly Size s_dropDownArrowSizeUnscaled = new(5, 3);
```